libs/chlorophyll/scss/components/alert-ribbon/_mixins.scss

```diff
@@ -102,12 +102,12 @@ $link-outline-color: hsla( var(--sg-intent-neutral-hsl-contrast), link.$outline-
       display: inline;
       font-size: 1rem;
       font-weight: 500;
-      @include common.line-height(3);
```

Comment:
Should we not use common.line-height any longer? 
If not, isn't it better if we remove it? Otherwise, we should use it here as we did before, but with correct rem size.

Reply:
For some reason there was 2 line-height scales and I wasn't sure which rule was written with which scale. So I decided to remove the use in this mixing and explicitly write the value so I don't get it wrong.

Then maybe if there is a need to use the scale - which I don't really see at this point - we can go back and use it... 

But yea, maybe remove it since we don't use it..